components/molecule/imageEditor/src/styles/settings.scss

```
@@ -1,2 +1,5 @@
 $h-molecule-image-editor: 300px !default;
 $w-molecule-image-editor-label-text: 50px !default;
+$gap-molecule-image-editor-rowLayout: 25px !default;
```

Use a variable (we have $g for gaps. Using variables also help to avoid numers that are not part of base 8px
```suggestion
$gap-molecule-image-editor-rowLayout: $g-xl !default;
```